bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/form/ReservedProperties.java

```diff
@@ -74,6 +74,7 @@ private ReservedProperties() {
     public static final String PN_PLACEHOLDER = "placeholder";
     public static final String PN_READ_ONLY = "readOnly";
     public static final String PN_DEFAULT_VALUE = "default";
+    public static final String PN_MULTI_DEFAULT_VALUES = "multiDefaultValues";
```

Comment:
This property should be prefixed with fd: right ?

Reply:
@rismehta Replied to this earlier: https://github.com/adobe/aem-core-forms-components/pull/1454#issuecomment-2426015289